moshi/src/main/java/com/squareup/moshi/ArrayJsonAdapter.java

```diff
@@ -69,4 +69,8 @@
     }
     writer.endArray();
   }
+
+  @Override public String toString() {
+    return "JsonAdapter(" + elementClass.getSimpleName() + ").array()";
```

Comment:
(because the elementClass doesn‘t tell as much as its adapter. I’m thinking things like `Date[]` vs. `Iso8601Adapter[]`